sotodlib/toast/sim_sso.py

```diff
@@ -118,6 +119,29 @@ def exec(self, data):
                 tmr.stop()
                 tmr.report("{}Simulated and observed SSO signal".format(prefix))
         return
+    
+    def _get_planet_temp(self,sso_name,freq):
+        """
+        Get the thermodynamic planet temperature given
+        the frequency
+        """
+        freqs = [30,44,70,100,143,217,353]
+
+        t_jupyter = [144.5, 159.1, 171.9, 172.6, 174.1, 175.8, 167.4]
+        t_saturn = [138.9, 147.3, 150.6, 145.7, 147.1, 145.1, 141.6]
+        t_mars = [183, 187, 183, 194.3, 198.4, 201.9, 209.9]
+        t_uranus = [190, 230, 138, 120.5, 108.4, 120.5, 98.5, 86.2]
```

Comment:
Just a reminder that the 44 GHz value for Uranus should be checked.  I suspect it's a typo.